workspace-loader/src/index.ts

```diff
@@ -44,7 +44,7 @@ export class KeycloakLoader {
                     if (request.status == 200) {
                         resolve(this.injectKeycloakScript(JSON.parse(request.responseText)));
                     } else {
-                        reject(null);
+                        reject('Cannot load keycloak script');
```

Comment:
reject(new Error('Cannot load keycloak script')
```
```suggestion
                        reject(new Error('Cannot load keycloak script'));
```